logger/LogManager.swift

```diff
@@ -139,9 +139,9 @@ class LogManager: ObservableObject {
                 sensor: IMUMeasurement(
                     type: .accelerometer,
                     values: [
-                        meas.acceleration.x,
-                        meas.acceleration.y,
-                        meas.acceleration.z])
+                        meas.acceleration.x * 9.8, // convert to mpss
+                        meas.acceleration.y * 9.8,
+                        meas.acceleration.z * 9.8])
```

Comment:
```suggestion
                        meas.acceleration.x * 9.81, // convert to mpss
                        meas.acceleration.y * 9.81,
                        meas.acceleration.z * 9.81])
```
I don't think that Kimera/Kalibr can correct scale errors. Seems like it's defined as 9.81 in the apple docs. https://developer.apple.com/documentation/coremotion/cmacceleration

Reply:
Good catch. I had pulled 9.8 from here: https://developer.apple.com/documentation/coremotion/getting-raw-accelerometer-events